docs/_snippets/actions-spyon-basic-example.md

```diff
@@ -0,0 +1,19 @@
+```ts filename=".storybook/preview.ts" renderer="common" language="ts"
+import { spyOn } from 'storybook/test';
+
+export default {
+  async beforeEach() {
+    spyOn(console, 'log').mockName('console.log');
+  },
+};
+```
+
+```js filename=".storybook/preview.js" renderer="common" language="js"
+import { spyOn } from 'storybook/test';
+
+export default {
+  async beforeEach() {
+    spyOn(console, 'log').mockName('console.log');
+  },
+};
+```
```

Comment:
@Sidnioulz shouldn't this be something along the lines of:
````suggestion
  ```ts filename=".storybook/preview.ts" renderer="common" language="ts"
// Replace your-framework with the framework you are using (e.g., react-vite, nextjs, svelte)
import type { Preview } from '@storybook/your-framework';

import { spyOn } from 'storybook/test';

const preview: Preview = {
  async beforeEach() {
    spyOn(console, 'log').mockName('console.log');
  },
};

export default preview;
```

```js filename=".storybook/preview.js" renderer="common" language="js"
import { spyOn } from 'storybook/test';

export default {
  async beforeEach() {
    spyOn(console, 'log').mockName('console.log');
  },
};
```
````